scripts/load_tester/start.py

```diff
@@ -0,0 +1,48 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv("../../.env")
+TX_SERVICE_URL = os.getenv("TRANSACTION_SERVICE_URL")
+PRINT_FORMAT = "{0:<10} {1:>8}::{2:>8}"
+
+
+def get_base_url() -> str:
+    if "staging" not in TX_SERVICE_URL:
+        return "https://safe-client.rinkeby.gnosis.io" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client.mainnet.gnosis.io"
+    else:
+        return "https://safe-client-rinkeby.staging.gnosisdev.com" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client-mainnet.staging.gnosisdev.com"
+
+
+def load_safes() -> list[str]:
+    print("Loading remote ...")
+    top_safes_url = TX_SERVICE_URL + "/api/v1/analytics/multisig-transactions/by-safe/?limit=300"
+    print(top_safes_url)
+    response = requests.get(top_safes_url)
+    return list(map(lambda safe: safe['safe'], response.json()['results']))
+
+
+base_gateway_url = get_base_url()
+safes = load_safes()
+print("Top 300 safes:")
+print("\n\t" + "\n\t".join(safes))
+print("Safes ready ... ")
+print("Populating cache ...")
+
+for safe in safes:
```

Comment:
A little bit inconsistent that this logic is not in a method while all the other logic is in methods

Reply:
do you mean the logic inside the `for`? Is that what you would have extracted, or do you want me to have a method called `main` with all the lose function calls ?

Reply:
I mean the whole for loop ;) ... not really important, it was more that the other to steps where in method so I thought it might be more consistent to have something like:
```python
base_gateway_url = get_base_url()
safes = load_safes()
load_information_for_safes(safes)
```